docs/book/src/faq.md

```diff
@@ -92,9 +92,38 @@ securityContext:
 ```
 However, note that this problem is fixed and will not occur if you deploy the project in high versions (maybe >= 1.22).
 
+## The error `Too long: must have at most 262144 bytes` is faced when I run `make install` to apply the CRD manifests . How to solve it? Why this error is faced?
+
+The Kubernetes API has a size limit. Then, when the apply is used the API will add the annotation last-applied-configuration with all previous configuration which is exceeding this amount. [More info][k8s-obj-creation]
+
+Therefore, you have a few options to workround this scenario such as:
+
+- You can use the option `maxDescLen=0` to remove the description of your CRDs. Following an example:
+
+```shell
+
+ .PHONY: manifests 
+ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects. 
+     # Note that the option maxDescLen=0 was added in the default scaffold in order to sort out the issue 
+     # Too long: must have at most 262144 bytes. By using kubectl apply to create / update resources an annotation 
+     # is created by K8s API to store the latest version of the resource ( kubectl.kubernetes.io/last-applied-configuration). 
+     # However, it has a size limit and if the CRD is too big with so many long descriptions as this one it will cause the failure. 
+ 	$(CONTROLLER_GEN) rbac:roleName=manager-role crd:maxDescLen=0 webhook paths="./..." output:crd:artifacts:config=config/crd/bases 
+
```

Comment:
```suggestion
```
```

Missing close the shell script. See that broke the doc.
You can always validate your changes by looking at the preview:
<img width="703" alt="Screenshot 2023-08-24 at 19 31 36" src="https://github.com/kubernetes-sigs/kubebuilder/assets/7708031/c2af4939-5e53-47b5-8a51-564e9472d343">

Just click in the details to see the whole doc rendered within the changes.
In this case, if you just either use github to check the doc via markdown you will see that is broke:
<img width="420" alt="Screenshot 2023-08-24 at 19 32 45" src="https://github.com/kubernetes-sigs/kubebuilder/assets/7708031/b4b7f921-1bda-4d3e-95fb-6419fd1a2123">
<img width="868" alt="Screenshot 2023-08-24 at 19 32 53" src="https://github.com/kubernetes-sigs/kubebuilder/assets/7708031/396cb321-d445-4b7d-a31c-f73fc4bda5ca">

Reply:
Missing close the shell script. See that broke the doc.

You can always validate your changes by looking at the preview:

<img width="703" alt="Screenshot 2023-08-24 at 19 31 36" src="https://github.com/kubernetes-sigs/kubebuilder/assets/7708031/c2af4939-5e53-47b5-8a51-564e9472d343">

Just click in the details to see the whole doc rendered within the changes.
In this case, if you just either use github to check the doc via markdown you will see that is broke:

<img width="420" alt="Screenshot 2023-08-24 at 19 32 45" src="https://github.com/kubernetes-sigs/kubebuilder/assets/7708031/b4b7f921-1bda-4d3e-95fb-6419fd1a2123">

<img width="868" alt="Screenshot 2023-08-24 at 19 32 53" src="https://github.com/kubernetes-sigs/kubebuilder/assets/7708031/396cb321-d445-4b7d-a31c-f73fc4bda5ca">